### PR TITLE
Various fixes to allow tutorials to work with astropy 3.1 (and python 3.6)

### PR DESCRIPTION
**Pull request by JuanCab**
I've been retesting the tutorial using the current astropy 3.1 and have found the following issues needing resolving:

1.  I am running into an error in **03-UnitsQuantities/Astropy_Units.ipynb**.  When I execute the cell containing:  `u.add_enabled_units([h100, h70])`  I am getting an error reported that `ValueError: Object with name 'littleh' already exists in namespace. Filter the set of units to avoid name clashes before enabling them.`  The error appears to exist because for some reason 'littleh' is treated as an existing unit. (Probably because it was added in astropy 3.1, incorrectly according to  https://github.com/astropy/astropy/issues/8271). Removing the declaration of the littleh unit with h100 seems to have fixed this.  If there is another solution out there, I would love to hear it.

2. Apparently `xlwt` wasn't caught as a required library to install in order to allow **06-Tables/Tables.ipynb** to run, specifically, the library is required by pandas to allow import of excel files.  I suspect the bug is a conda bug in that the requirement should have been caught.  Fixed the `environment.yml` file to include this in its list of required packages.

3. Added `imexam` to `environment.yml` file to so that it will be available for the **09-Photutils/image_psf_photometry_withNIRCam.ipynb** notebook if needed.  I have confirmed that the `environment.yml` setup works for macOS and windows-64bit using conda and linux using miniconda.

4. I removed superfluous "from __future__ import print_function, division" lines from various notebooks since we are forcing use of python 3.6 in the `environment.yml` file.  This resolves issue #25 I believe.

5. Removed notebook output accidentally included in repo in **04-Coordinates/astropy_coordinates.ipynb**.

6. Updated `check_env.py` in the **00-Install_and_Setup** directory to check out properly on macos/windows/linux under the new `environment.yml` setup.

Once these issues are resolved, everything will run in the `astropy-workshop` environment with astropy 3.1 EXCEPT that the `imexam` example in **09-Photutils/image_psf_photometry_withNIRCam.ipynb** will throw an error for reasons cited in issue #29 .

### CODE REVIEW

**Comment by JuanCab** [comment]
I discovered another small glitch, the `check_env.py` script to check the environment was throwing an error on windows (since we now use a normal installation of `imexam` from astropy instead of a pip install).  I have updated the script to match the current installation expectations.

**Comment by sosey** [comment]
@JuanCab I think we should update the check_env script to match the environment.yaml versions. Example: we had set the matplotlib version to the LTS 2.2.3, but checkenv goes lower. Also, we should update imexam to 0.8.1.

File: 00-Install_and_Setup/check_env.py [hunk 1]
```diff
@@ -36,22 +36,20 @@ def check_package(package_name, minimum_version=None, verbose=True):
...
-        'matplotlib': '1.3',
-        'astropy': '1.2.1',
-        'photutils': '0.4',
-        'skimage': '0.12.3',
-        'pandas': '0.18.1',
+pkgs = {'IPython': '7.2',
+        'notebook': '5.7',
+        'numpy': '1.14',
+        'scipy': '1.1',
+        'matplotlib': '2.0',
```
Comment (sosey) [inline 00-Install_and_Setup/check_env.py]: still need the 2.2.3 version of matplotlib here

Comment (JuanCab) [reply 00-Install_and_Setup/check_env.py]: Just addressed it.

**Pull request merged**